runtime/isp_qemu.py

```diff
@@ -90,11 +90,6 @@ def launchQEMU(exe_path, run_dir, policy_dir, runtime, extra, use_validator=True
                     rc.terminate()
                     process_exit = True
                     return
-                if "Policy Violation:" in open(os.path.join(run_dir, status_log_file), 'r').read() or process_exit:
```

Comment:
I don't think this is needed in FRTOS/Qemu... don't know whether it would still be needed for bare metal? If so, easy enough to support by conditioning on the runtime, which we already have in this fn.

Reply:
We might want to parameterize this check instead of removing it (I assume this prevents a FreeRTOS process from entering its policy violation handler)

Reply:
We can parameterize if bare metal needs it but if bare metal has test finisher support then we can take it out, as that is clearly the “right” way to do it. 

In fact, we should also have `test_end()` call the test finisher so we don’t need that second log file monitor either.

Reply:
That's fair. Might make sense to re-do exit handling for QEMU in a separate PR

Reply:
`test_done` already calls the test finisher.  It doesn't seem like either log-checking statements are needed anymore.